README.rst

```diff
@@ -523,8 +525,6 @@ Choose from the list of available configurations:
 The ``--dry-run`` option displays the files that need to be
 fixed but without actually modifying them:
 
-.. code-block:: bash
```

Comment:
@keradus When I run `php php-cs-fixer readme > README.rst` this is what it returns. Any ideas how to fix this?